### PR TITLE
Hardsuit Jetpack Upgrade

### PR DESCRIPTION
**Pull request by GunHog**
Replaces the mining jetpack with the hardsuit jetpack upgrade. This is intended to allow miners to actually enjoy their hard-earned jetpack without having to juggle their inventories.

This is an item that will install a jetpack into a hardsuit when used. Acquired for 2,000 mining points from the equipment vendor.

Sprite by @WJohn!
upgradejetpack

### CODE REVIEW

File: code/modules/mining/equipment_locker.dm [hunk 1]
```diff
@@ -885,15 +885,25 @@
...
-	item_state = "jetpack-mining"
-	desc = "A tank of compressed carbon dioxide for miners to use as propulsion in local space. The compact size allows for easy storage at the cost of capacity."
-	volume = 40
-	throw_range = 7
-	w_class = 3 //same as syndie harness
+/******************Hardsuit Jetpack Upgrade*******************/
+/obj/item/hardsuit_jetpack
+	name = "hardsuit jetpack upgrade"
+	icon_state = "jetpack_upgrade"
+	desc = "A modular, compact set of thrusters designed to intigrate with a hardsuit. It is fueled by a tank inserted into the suit's storage compartment."
```
Comment (GunHog) [inline code/modules/mining/equipment_locker.dm]: Blasted spelling mistakes...

**Comment by xxalpha** [comment]
Good change.

I give it a less than a month until someone is adding these to the armory or R&D.

**Comment by GunHog** [comment]
While I disagree with your intent to permanently deny any additional content to Science, I will not be the one responsible. I cannot at all speak for Security players.

**Comment by TheNightingale** [comment]
As a Security player, we don't really need these in the Armory. Sure, Security could use jetpacks on their hardsuits, but they're not 100% necessary - we can just ask Mining (miners have thousands of points when the ORM is upgraded).

**Comment by optimumtact** [comment]
Yeah if anything this should be a very expensive item.
Right now I can buy basically everything worth buying in one mining run and still have plenty to buy stims for anyone who wants to run fast.

**Comment by TheNightingale** [comment]
All the good Mining gear is in R&D anyway (sonic jackhammer, bag of holding, hyper-kinetic accelerator, upgraded resonator, mining satchel of holding). What if we moved some of the good R&D gear... over to the Mining vendors? That way, Mining can bring along some stuff R&D can deconstruct as well as the minerals.

**Comment by GunHog** [comment]
Miners are one of the few player types that bother to drop by Science for items, doing that would remove their reason for coming over. I like talking to the miners, they are some of the most helpful and usually pleasant players I have seen.

Anyway, that is outside the scope of the PR, please keep discussion to the item in question. @optimumtact, what price would you suggest? I am open to raising it a bit, but nothing extreme.

**Comment by optimumtact** [comment]
ah just leave it, it'd need to be done as a proper rebalancing of all the points

**Comment by KorPhaeron** [comment]
God forbid anyone ever have to make any choice in what they are going to carry or that max speed flight in space has a tradeoff. Hardsuit jetpacks should deploy like the helmet and take up the bag slot if they must continue to exist.

**Comment by xxalpha** [comment]
The whole point of hardsuit jetpacks was to not use the back slot.

**Comment by KorPhaeron** [comment]
I do not believe that is a good goal because

1) There has been a steady trend of letting people carry more and more shit, which kills reliance on other players and interaction with them

2) Jetpack is one of the strongest items in the game. Them having a drawback is good

**Comment by kevinz000** [comment]
I find it silly that an engineer is better at space combat then security armoury hard suits.

**Comment by xxalpha** [comment]
The drawback is that it needs a gas tank in the suit slot.

**Comment by KorPhaeron** [comment]
That is not a drawback at all. You can just shove whatever was going to be in your suit slot into your bag (along with half a dozen other things)

**Comment by KazeEspada** [comment]
Why do miners have to pay for what atmos techs get for free?

**Comment by xxalpha** [comment]
You can't use internals from the bag!

**Comment by KorPhaeron** [comment]
So you put your internals in your pocket? A backpack > a pocket.

**Comment by xxalpha** [comment]
Those are smaller tanks, though.

**Comment by Lumipharon** [comment]
And they also last for 30minutes plus, if it's the extended emergency tank.

Also can't you jsut like, use the suit tank for the jetpack AND breathing anyway?

**Comment by xxalpha** [comment]
You can but the jetpack does use up the gas pretty fast.

**Comment by PKPenguin321** [comment]
>Also can't you jsut like, use the suit tank for the jetpack AND breathing anyway?

yep, and since you can hold like 4 tanks in your backpack it's hardly ever an issue

**Comment by KorPhaeron** [comment]
Small tanks last ages when filled. And you can store spares in your internals box.

I don't know why we're having this argument. If you didnt believe that freeing up the backslot was an advantage you wouldnt have bothered adding the inbuilt hardsuits in the first place.

**Comment by Lumipharon** [comment]
^ jetpacks are grossly strong, it's already not an unusual sight to see a miner with a jetpack and a KA utterly WRECKING multiple nuke ops.

**Comment by xxalpha** [comment]
I never said it wasn't an advantage, I specifically said its their purpose.

I admit a retractable jetpack is better than handling the item itself.

**Comment by Lumipharon** [comment]
It creeeeeps

**Pull request merged**